modules/kubernetes-addons/kyverno/main.tf

```diff
@@ -0,0 +1,35 @@
+resource "kubernetes_namespace_v1" "this" {
+  metadata {
+    name = local.kyverno_helm_config["namespace"]
+    labels = {
+      "app.kubernetes.io/managed-by" = "terraform-aws-eks-blueprints"
+    }
+  }
+}
+module "kyverno_helm_addon" {
+  count             = var.enable_kyverno ? 1 : 0
```

Comment:
this should be created regardless - we can't have a reporter or baselines without kyverno itself